src/config.js

```diff
@@ -43,7 +43,7 @@ const config = {
       title: "Deforestation: 48 Years of Kenya's Unspoken Disaster",
       author: 'Africa Uncensored',
       brief: 'Deforestation in Kenya, how did it happen? Who is responsible?',
-      link: '',
+      link: 'https://www.youtube.com/embed/PDhuK4KrIoU',
```

Comment:
```suggestion
      link: 'https://www.youtube.com/watch?v=PDhuK4KrIoU',
```

Reply:
@DavidLemayian This link?  https://africauncensored.online/

Reply:
No, the link I've suggested.